miden-tx/tests/integration/scripts/p2idr.rs

```diff
@@ -199,14 +198,14 @@ fn p2idr_script() {
     let note_ids_4 = data_store_4.notes.iter().map(|note| note.id()).collect::<Vec<_>>();
 
     // Execute the transaction and get the witness
-    let executed_transaction_4 = executor_4
-        .execute_transaction(target_account_id, block_ref_4, &note_ids_4, Some(tx_script_target))
+    let transaction_result_4 = executor_4
```

Comment:
I don't think we need to change `executed_transaction` into `transaction_result` here and in other places in this file.